keyboards/kprepublic/bm40hsrgb/keymaps/wolff_abnt2/rules.mk

```diff
@@ -0,0 +1,25 @@
+MCU = atmega32u4
+BOOTLOADER = atmel-dfu
+EXTRAFLAGS += -flto
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+TERMINAL_ENABLE = no
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+NKRO_ENABLE = no            # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+AUDIO_ENABLE = no           # Audio output
+SPLIT_KEYBOARD = no
+KEY_LOCK_ENABLE = no
+RGB_MATRIX_ENABLE = yes
+RGB_MATRIX_DRIVER = WS2812
+LAYOUTS = planck_mit
+LAYOUTS_HAS_RGB = yes
+
+MIDI_ENABLE = yes
+JOYSTICK_ENABLE = yes
+JOYSTICK_DRIVER = digital
```

Comment:
Your keymap has nothing to do with joystick/pointing devices as far as i can see, so this can be removed (part 2)
```suggestion
```

Reply:
See previous discussion.